packages/office-addin-dev-certs/src/httpsServerOptions.ts

```diff
@@ -14,6 +15,12 @@ export async function getHttpsServerOptions(): Promise<IHttpsServerOptions> {
     await ensureCertificatesAreInstalled();
 
     const httpsServerOptions = {} as IHttpsServerOptions;
+    try {
+        httpsServerOptions.ca = fs.readFileSync(defaults.localhostCaCertificatePath);

```

Comment:
Should just be `defaults.caCertificatePath` once the duplicative `defaults.localhostCaCertificatePath` is removed.

Reply:
caCertifatePath resulted in test failures. Found I had to add this new default

Reply:
Aren't these the same? It would be better to fix the tests that introduce this. Sachin can help with the test failures...

Reply:
I guess I don't need to include localhostCaCertificateFileName and localhostCaCertificatePath. I could've sworn I was getting a test failure without this change but must have been some other reason. Removing them

Reply:
Removing them